src/D04/Pair.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace aoc.D04
+{
+  public struct Pair
```

Comment:
@taherkheli you could use  the new record type :D it has some nice properties like equality comparison